bigquery/system-test/tables.test.js

```diff
@@ -0,0 +1,79 @@
+// Copyright 2016, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+var uuid = require('node-uuid');
+var generateUuid = function () {
+  return 'nodejs_docs_samples_' + uuid.v4().replace(/-/gi, '_');
+};
+var example = require('../tables');
+var options = {
+  bucket: generateUuid(),
+  file: 'data.json',
+  dataset: generateUuid(),
+  table: generateUuid()
+};
+var BigQuery = require('@google-cloud/bigquery');
+var bigquery = BigQuery();
+var Storage = require('@google-cloud/storage');
+var storage = Storage();
+var file = storage.bucket(options.bucket).file(options.file);
+
+describe('bigquery:tables', function () {
+  before(function (done) {
+    // Create bucket
+    storage.createBucket(options.bucket, function (err, bucket) {
+      assert.ifError(err, 'bucket creation succeeded');
+
+      // Create dataset
+      bigquery.createDataset(options.dataset, function (err, dataset) {
+        assert.ifError(err, 'dataset creation succeeded');
+
+        // Create table
+        dataset.createTable(
+          options.table,
+          { schema: 'name:string, age:integer' },
+          function (err, table) {
+            assert.ifError(err, 'table creation succeeded');
+            done();
+          }
+        );
+      });
+    });
+  });
+  after(function (done) {
+    // Delete table export
+    file.delete(function () {
+      // Delete testing dataset/table
+      bigquery.dataset(options.dataset).delete({ force: true }, done);
+    });
+  });
```

Comment:
This test isn't deleted the bucket that it creates. The simplest way to clean up a bucket and all its files can be found here: https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/storage/system-test/files.test.js#L41